lesson8/src/two/Wall.java

```diff
@@ -3,9 +3,7 @@
 public class Wall implements Barrier {
 
     private int height;
-
     private String name;
-
     public Wall(int height, String name) {
         this.height = height;
```

Comment:
all files from the top to this message should be removed, they are not related to homework number 10